src/sourmash/minhash.py

```diff
@@ -684,6 +684,22 @@ def to_frozen(self):
         new_mh.__class__ = FrozenMinHash
         return new_mh
 
+    def inflate(self, from_mh):  
+        if (self.track_abundance == False) and (from_mh.track_abundance == True):
+            hashes = set(self.hashes)
+            orig_abunds = from_mh.hashes
+            abunds = { h: orig_abunds[h] for h in hashes }
+
+            abund_query_mh = from_mh.copy_and_clear()
```

Comment:
suggest changing this to `abund_mh`.

Reply:
So, `abund_query_mh` should now be `abund_mh` in all places right?

Reply:
what is the alternative?